include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -504,12 +504,12 @@ class argument_parser
                 std::string export_format{argv[i+1]};
 
                 if (export_format == "html")
-                    format = detail::format_html{};
+                    format = detail::format_html{true};
                 else if (export_format == "man")
-                    format = detail::format_man{};
+                    format = detail::format_man{true};
```

Comment:
what is being set to true here?

Reply:
The `show advanced options`

Reply:
Hm, that's kind of obscure. If they always show advanced, you shouldn't have to pass this parameter.

Reply:
Well, they use the  `add_option` call from the base class which only adds a list item for advanced options if the spec is set. I would need to reimplement the 3 functions for html, man and ctd format then which I explicitly tried to avoid since they do the same thing. So this is no additional line of code versus +90 LOC.

Reply:
You could just default-init it to true and not provide a constructor with the argument, except for format_help which is the only one which needs to edit it...